Extension/i18n/kor/src/LanguageServer/settings.i18n.json

```diff
@@ -4,5 +4,5 @@
  *--------------------------------------------------------------------------------------------*/
 // Do not edit this file. It is machine generated.
 {
-	"editorconfig.default.behavior": "코드 형식이 .clang 형식 대신 .editorconfig의 설정을 사용합니다. 자세한 내용은 'C_Cpp.formatting' 설정의 'Default' 값에 대한 설명서를 참조하세요."
+	"editorconfig.default.behavior": "코드 형식이 .clang 형식 대신 .editorconfig의 설정을 사용합니다. 자세한 내용은 'C_Cpp.formating' 설정의 'Default' 값에 대한 설명서를 참조하세요."
```

Comment:
Not sure why someone updating this string just to misspell this setting name.  Please don't commit until we get this corrected.

Reply:
I filed a bug on that already.